dll/win32/imm32/imm.c

```diff
@@ -1161,6 +1161,55 @@ static PCLIENTIMC APIENTRY Imm32GetClientImcCache(void)
     return NULL;
 }
 
+static BOOL Imm32IsValidHimc(HIMC hIMC)
+{
+    return (hIMC != NULL); // FIXME
```

Comment:
If you give imm32 it's own heap for `Imm32HeapAlloc`, and you allocate an HIMC from this heap,
you can use `RtlValidateHeap` to check that it was allocated from the correct heap.
That seems like a good starting point to validate an HIMC?

Reply:
OK. Thanks.